src/Esfa.Vacancy.UnitTests/Shared/Infrastructure/GivenStructureMapRegistry.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Esfa.Vacancy.Domain.Constants;
+using Esfa.Vacancy.Domain.Interfaces;
+using Esfa.Vacancy.Infrastructure;
+using Esfa.Vacancy.Infrastructure.Services;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.NLog.Logger;
+using StructureMap;
+
+namespace Esfa.Vacancy.UnitTests.Shared.Infrastructure
+{
+    public class GivenStructureMapRegistry
+    {
+        [Test]
+        public void AndIsReadonlyEnabled()
```

Comment:
Is it worth renaming the method to AndUseSandboxServiceEnabled ?

Reply:
fixed